pkg/rke/templates/templates.go

```diff
@@ -117,8 +118,10 @@ func LoadK8sVersionedTemplates() map[string]map[string]string {
 	return map[string]map[string]string{
 		kdm.Calico: {
 			">=1.27.0-rancher1-1":                      calicov3261,
-			">=1.26.0-rancher1-1 <1.27.0-rancher1":     calicov3250,
-			">=1.25.0-rancher1-1 <1.26.0-rancher1-1":   calicov3241,
+			">=1.26.11-rancher1 <1.27.0-rancher1":      calicov3261,
```

Comment:
`">=1.27.0-rancher1-1"` and `">=1.26.11-rancher1 <1.27.0-rancher1"` both are for  `calicov3261`, could you update this to `">=1.26.11-rancher1"`?

Reply:
updated